<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="AKM_Laptop" failures="0" tests="6" name="com.TestNG.Oct_29_Retry_Failed_TestCases.LoginTest" time="26.030" errors="1" timestamp="2024-01-22T15:31:34 EST" skipped="5">
  <testcase classname="com.TestNG.Oct_29_Retry_Failed_TestCases.LoginTest" name="loginValidation" time="4.145">
    <skipped/>
  </testcase> <!-- loginValidation -->
  <testcase classname="com.TestNG.Oct_29_Retry_Failed_TestCases.LoginTest" name="loginValidation" time="3.884">
    <skipped/>
  </testcase> <!-- loginValidation -->
  <testcase classname="com.TestNG.Oct_29_Retry_Failed_TestCases.LoginTest" name="loginValidation" time="4.601">
    <error message="element not interactable
  (Session info: chrome=120.0.6099.217)
Build info: version: &#039;4.16.1&#039;, revision: &#039;9b4c83354e&#039;
System info: os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;17.0.7&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [c474c7ea90096b8fed78932be9734633, sendKeysToElement {id=9E3C87423908C64873ADEF8993D50D7B_element_542, value=[Ljava.lang.CharSequence;@53a5e217}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 120.0.6099.217, chrome: {chromedriverVersion: 120.0.6099.109 (3419140ab66..., userDataDir: C:\Users\misfo\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:62161}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:62161/devtoo..., se:cdpVersion: 120.0.6099.217, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (c474c7ea90096b8fed78932be9734633)] -&gt; css selector: #content &gt; div &gt; div:nth-child(2) &gt; div &gt; form &gt; div:nth-child(2)]
Session ID: c474c7ea90096b8fed78932be9734633" type="org.openqa.selenium.ElementNotInteractableException">
      <![CDATA[org.openqa.selenium.ElementNotInteractableException: element not interactable
  (Session info: chrome=120.0.6099.217)
Build info: version: '4.16.1', revision: '9b4c83354e'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [c474c7ea90096b8fed78932be9734633, sendKeysToElement {id=9E3C87423908C64873ADEF8993D50D7B_element_542, value=[Ljava.lang.CharSequence;@53a5e217}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 120.0.6099.217, chrome: {chromedriverVersion: 120.0.6099.109 (3419140ab66..., userDataDir: C:\Users\misfo\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:62161}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:62161/devtoo..., se:cdpVersion: 120.0.6099.217, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (c474c7ea90096b8fed78932be9734633)] -> css selector: #content > div > div:nth-child(2) > div > form > div:nth-child(2)]
Session ID: c474c7ea90096b8fed78932be9734633
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:52)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:200)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:175)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:523)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:224)
at org.openqa.selenium.remote.RemoteWebElement.sendKeys(RemoteWebElement.java:111)
at com.TestNG.Oct_29_Retry_Failed_TestCases.LoginTest.loginValidation(LoginTest.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:249)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- loginValidation -->
  <testcase classname="com.TestNG.Oct_29_Retry_Failed_TestCases.LoginTest" name="loginValidation" time="4.691">
    <skipped/>
  </testcase> <!-- loginValidation -->
  <testcase classname="com.TestNG.Oct_29_Retry_Failed_TestCases.LoginTest" name="loginValidation" time="4.311">
    <skipped/>
  </testcase> <!-- loginValidation -->
  <testcase classname="com.TestNG.Oct_29_Retry_Failed_TestCases.LoginTest" name="loginValidation" time="4.398">
    <skipped/>
  </testcase> <!-- loginValidation -->
</testsuite> <!-- com.TestNG.Oct_29_Retry_Failed_TestCases.LoginTest -->
